### PR TITLE
TMA-1100 fix productized connector deployment

### PR DESCRIPTION
**Pull request by krystofspl**
TMA-1100 fix productized connector deployment

### CODE REVIEW

File: lib/gooddata/models/process.rb [hunk 1]
```diff
@@ -187,8 +187,8 @@ def deploy_from_appstore(path, options = { :client => GoodData.client, :project
       def deploy_component(data, options = { client: GoodData.client, project: GoodData.project })
         client, project = GoodData.get_client_and_project(options)
         data = { process: data } unless data[:process]
-        data[:process] = GoodData::Helpers.symbolize_keys(data[:process]).select { |k| [:type, :name, :component].include? k }
-        data[:process][:component] = GoodData::Helpers.symbolize_keys(data[:process][:component]).select { |k| [:name, :version].include? k }
+        data[:process] = GoodData::Helpers.symbolize_keys(data[:process]).select { |k| %i[type name component].include? k }
```
Comment (yenkins) [inline lib/gooddata/models/process.rb]: info: Calls 'data[:process]' 4 times (DuplicateMethodCall) (reek)

Comment (yenkins) [inline lib/gooddata/models/process.rb]: info: Has the variable name 'k' (UncommunicativeVariableName) (reek)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/751/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1573/ : SUCCESS in 2m 52s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1834/ : FAILURE in 1m 50s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/14/ : SUCCESS in 7m 42s

**Comment by krystofspl** [comment]
Merged in #1324.

**Pull request closed without merging**